deliver/lib/deliver/setup.rb

```diff
@@ -67,6 +67,18 @@ def generate_metadata_files(v, path)
       end
 
       UI.success("Successfully created new configuration files.")
+
+      # get App icon + watch icon
+      if v.large_app_icon.asset_token
+        app_icon_path = File.join(path, "app_icon.png")
+        File.write(app_icon_path, open(v.large_app_icon.url).read)
```

Comment:
So if there is no app icon available yet, the `if` block before would detect that, right?